composer.json

```diff
@@ -21,11 +21,11 @@
         }
     ],
     "require": {
-        "php": "^7.3 || ^8.0",
+        "php": "^8.2",
```

Comment:
lets do ^8.1 we don't need the very latest and symfony 6.1 allows 8.1

Reply:
yeah but i switch to symfony 7 and symfony 7 needs php8.2 no ?

Reply:
Ok. I didn't know that. Thanks.